Content.Server/GameTicking/Rules/Configurations/GenericGameRuleConfiguration.cs

```diff
@@ -0,0 +1,14 @@
+﻿using JetBrains.Annotations;
+
+namespace Content.Server.GameTicking.Rules.Configurations;
+
+/// <summary>
+/// A generic configuration, for game rules that don't have special config data.
+/// </summary>
+[UsedImplicitly]
+public sealed class GenericGameRuleConfiguration : GameRuleConfiguration
+{
+    [DataField("id", required: true)]
+    private string _id = default!;
+    public override string Id => _id;
```

Comment:
wut

Reply:
tl;dr I didn't want to make a new class for every single game rule that doesn't already have a config, and you can't override fields.